epictrack-web/src/components/shared/index.tsx

```diff
@@ -438,6 +438,13 @@ export const ETPreviewText = styled(Typography)(() => ({
   lineHeight: "21px",
 }));
 
+export const ETStatusHistoryDate = styled(Typography)(() => ({

```

Comment:
Should this be a shared component. It can be one of the components inside the status folder